### PR TITLE
model relationships setup and tested

### PR DESCRIPTION
**Pull request by perryr16**
## Generate models for 
* Game
* User 
* FearQuiz
* FearRating
* Review

### CODE REVIEW

File: db/migrate/20200721230149_create_games.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class CreateGames < ActiveRecord::Migration[5.2]
+  def change
+    create_table :games do |t|
+      t.string :title
+      t.integer :age_rating
+      t.float :agg_rating
+      t.text :artworks, array: true, default: []
```
Comment (perryr16) [inline db/migrate/20200721230149_create_games.rb:7]: A lot of the values returned from the API are in array format. This syntax should allow us to shovel values in

File: db/migrate/20200721230149_create_games.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      t.float :rating
+      t.integer :rating_count
+      t.text :release_date, array: true, default: []
+      t.text :screenshots, array: true, default: []
+      t.text :similar_games, array: true, default: []
+      t.string :storyline
+      t.text :themes, array: true, default: []
+      t.float :total_rating
+      t.string :url
+      t.string :video
```
Comment (perryr16) [inline db/migrate/20200721230149_create_games.rb:27]: This is far more columns than we discussed but I thought we could remove them if we dont need them

File: spec/factories/games.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+FactoryBot.define do
+  factory :game do
```
Comment (perryr16) [inline spec/factories/games.rb:2]: Automatically generated factory bot models

File: db/migrate/20200721224611_create_users.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :google_token
+      t.string :email
+      t.string :photo, default: 'https://ftnj.com/wp-content/uploads/2018/09/female-headshot-silhouette.jpg'
```
Comment (perryr16) [inline db/migrate/20200721224611_create_users.rb:7]: default photo is a female silhouette, I think it looks a bit gender nuetral. I'd love to have that hair
[](https://ftnj.com/wp-content/uploads/2018/09/female-headshot-silhouette.jpg)

Comment (whitneykidd) [reply db/migrate/20200721224611_create_users.rb:7]: ooh, nice find on this one. looks a bit more 'modern' than the standard myspace default

File: db/migrate/20200721230149_create_games.rb (hunk 1 above)
Comment (whitneykidd) [reply db/migrate/20200721230149_create_games.rb:7]: Shoveling in values is a pretty clever move. 💯

File: db/migrate/20200721224611_create_users.rb (hunk 4 above)
Comment (melatran) [reply db/migrate/20200721224611_create_users.rb:7]: I like this default photo. That hair looks great :)

File: db/migrate/20200721230149_create_games.rb (hunk 2 above)
Comment (melatran) [reply db/migrate/20200721230149_create_games.rb:27]: We have lots of ideas on what to implement so totally makes sense that we just remove what we don't need.

**Pull request merged**